packages/salesforcedx-vscode-core/test/vscode-integration/context/workspaceOrgType.test.ts

```diff
@@ -136,8 +137,117 @@ describe('getWorkspaceOrgType', () => {
   });
 });
 
+describe.only('setWorkspaceOrgTypeWithOrgType', () => {
```

Comment:
These tests don't fully pass anymore.  They were written before I renamed enableOrgBrowser() to setEnableOrgBrowser(), and added enableOrgBrowser().  They just need a little cleanup and I will be working on them later today.